demo/build.gradle

```diff
@@ -18,7 +18,7 @@ dependencies {
 }
 
 android {
-    compileSdkVersion 23
+    compileSdkVersion "Google Inc.:Google APIs:21"
```

Comment:
Actually, I figured out a way to get this to run on API 21 emulator via the travis script itself.  I'll force-push a new commit into this PR that keeps the project `compileSdkVersion 23` while running on the API Level 21 Google API emulator.